src/applications/lgy/coe/form/content/LoggedInContent.jsx

```diff
@@ -37,7 +36,7 @@ const LoggedInContent = ({ route }) => (
         headingLevel={2}
       />
     </div>
-    <OMBInfo expDate="11/30/2022" ombNumber="2900-0086" resBurden={15} />
+    <va-omb-info exp-date="11/30/2022" omb-number="2900-0086" res-burden={15} />
```

Comment:
does the component `<va-omb-info/>` this automatically get imported?

Reply:
All web components (components starting with `va-`) get imported by the main component (the one that sets up Redux)